coala_quickstart/generation/Utilities.py

```diff
@@ -13,3 +16,73 @@ def comma_split(response):
             result.append(item)
 
     return result
+
+
+def is_glob_exp(line):
+    """
+    Determines whether a the string is a result.append(expression.
+
+    :param line:
+        Given string.
+    :return:
+        True if the string is a result.append(expression. False otherwise.
+    """
+    glob_chars = "*!?[]"
+    is_glob = False
+
+    index = 0
+    while index < len(line):
+        if line[index] == '\\':
+            index += 2
+            continue
+        if line[index] in glob_chars:
+            is_glob = True
+        index += 1
+
+    return is_glob
+
+
+def get_gitignore_glob(project_dir):
+    """
+    Generates a list of result.append(expressions equivalent to the
+    contents of the user's project's .gitignore
+
+    :param project_dir:
+        The user's project directory.
+    :return:
+        A list of result.append(expressions generated from the
+        ``.gitignore`` file.
+    """
+    result = []
+    gitignore = os.path.join(project_dir, ".gitignore")
+
+    with open(gitignore, "r") as file:
```

Comment:
please seperate reading from a concrete file and parsing it in seperate function, this smells